tests/functional/u/useless/useless_else_on_loop.py

```diff
@@ -62,9 +62,9 @@ def test_break_in_orelse_deep():
     """no false positive for break in else deeply nested
     """
     for _ in range(10):
-        if 1 < 2:
+        if 1 < 2: # pylint: disable=comparison-of-constants
```

Comment:
This file is also missing double spacing.

Reply:
black doesn't seem to mind about this issue; even if I do `pre-commit run --all-files`, the double space doesn't get put in. Does this depend on some kind of configuration, or?

Reply:
No, `pre-commit` sadly won't fix this for us. We explicitly exclude these test files from `pre-commit`: for example, sometimes we want to test that even if there are mistakes our syntactic errors in code `pylint` exits gracefully instead of crashing. We don't want `pre-commit` to warn us about such "mistakes".

Thus, keeping style in these files is a little harder. Many of the contributors and reviewers have set up their IDE to also run some formatters whenever they save a file. For example, I run `black` on save together with an import sorter. This helps keep the style in our test files somewhat similar. However, when people with such "on-save-formatters" look at these test files and make a change their formatters will also change stuff unrelated to the issue they are working on, as the formatter might start sorting imports or formatting lines. So, keeping the general style in our test files not only is a stylistic choice but also helps keep some of our contributors more productive. It's a bit annoying that we need to do this by hand, especially for larger PRs such as this one, but in the long run it tends to make us all more productive 😄

Reply:
Oh I get it now. `pre-commit` won't do this, by deliberate design. But of course we I could invoke black on those files ourselves.

Anyway I've done it manually

Reply:
Those file are sometime not black formatted by design, if we run black everywhere we won't have any tests for single quote strings for example. I think we should go easy on the normalization, we want to also catch bugs for users that do not use black or isort or do not respect pep8 to the letter, it's a good thing if the functional tests are somewhat messy.